templates/tls.cnf

```diff
@@ -0,0 +1,6 @@
+[DEFAULT]
+client_ssl_mode=REQUIRED
+client_ssl_key=$tls_ssl_key_file
+client_ssl_cert=$tls_ssl_cert_file
+
+# vim: filetype=ini
```

Comment:
nit: is this line necessary?

Reply:
not at all, just force my editor to correctly syntax highlight the file